src/gui/shellwidget/shellcontents.cpp

```diff
@@ -58,28 +58,27 @@ ShellContents::ShellContents(const ShellContents& other)
 /// needed.
 void ShellContents::allocData()
 {
-	_data = new Cell[_rows*_columns];
+	_data = new Cell[_rows * _columns];
 }
 
 void ShellContents::clearAll(QColor bg)
 {
-	for (int i=0; i<_rows; i++) {
-		for (int j=0; j<_columns; j++) {
-			_data[i*_columns+j] = Cell::bg(bg);
+	for (int i = 0; i < _rows; i++) {
```

Comment:
Cannot find an option for clang-format to disable space around operators.

Reply:
I;ll test SpaceBeforeAssignmentOperators to false